pyproject.toml

```diff
@@ -7,27 +7,29 @@ name = "blop"
 description = "Beamline optimization with machine learning"
 readme = { file = "README.rst", content-type = "text/x-rst" }
 authors = [
-  { name = "Brookhaven National Laboratory", email = "tmorris@bnl.gov" }
+  { name = "Brookhaven National Laboratory", email = "tmorris@bnl.gov" },
 ]
 maintainers = [
-  { name = "Brookhaven National Laboratory", email = "tmorris@bnl.gov" }
+  { name = "Brookhaven National Laboratory", email = "tmorris@bnl.gov" },
 ]
 requires-python = ">=3.9"
 
 dependencies = [
-    "area-detector-handlers",
-    "bluesky",
-    "botorch",
-    "databroker",
-    "gpytorch",
-    "h5py",
-    "matplotlib",
-    "numpy",
-    "ophyd",
-    "ortools",
-    "scipy",
-    "tables",
-    "torch",
+  "area-detector-handlers",
+  "bluesky",
+  "bluesky-adaptive[all]",
+  "botorch",
+  "databroker",
+  "gpytorch",
+  "h5py",
+  "matplotlib",
+  "numpy<2",
```

Comment:
I'm assuming you ran into an issue with `numpy>2`?

Reply:
No, I messed up the merge-conflict during the rebase. Dropped the "<2".